style/desktop.scss

```diff
@@ -1,52 +1,74 @@
 @media only screen and (min-width: 992px) {
+  body {
+    margin: auto;
+    width: 1260px;
+  }
 
-    body{
-        margin: auto;
-        width: 1260px;
-    }
-    
-    /********************************HEADER****************************/
-    
-    header{ 
-      .svg-logo{
-        width: 187px;
-        height: 187px;
+  /********************************HEADER****************************/
+
+  header {
+    .svg-logo {
+      width: 187px;
+      height: 187px;
     }
-    
-      button{
-        display: none;
-      }
+
+    button {
+      display: none;
     }
-    
-    div.ham-btn-menu{
-      visibility: visible;
+  }
+
+  div.ham-btn-menu {
+    visibility: visible;
+    position: absolute;
+    top: 0;
+    background-color: transparent;
+    width: 100%;
+    height: 0px;
+
+    ul {
       position: absolute;
       top: 0;
-      background-color: transparent;
-      width: 100%;
-      height: 0px;
-      
-      ul{
-        position: absolute;
-        top: 0;
-        padding: 0;
-        margin-left: 290px;
-        margin-top: 60px;
-      }
-      li{
-        display: inline;
-        margin-left: 50px;
-        button{
-          height: 45px;
-        }
-      }
-      a{
-        font-size: 1.5em;
-      }
-      button{
-        height: 34px;
-        margin-left: 70px;
-      
+      padding: 0;
+      margin-left: 290px;
+      margin-top: 60px;
+    }
+    li {
+      display: inline;
+      margin-left: 50px;
+      button {
+        height: 45px;
       }
     }
- }
\ No newline at end of file
+    a {
+      font-size: 1.5em;
+    }
+    button {
+      height: 34px;
+      margin-left: 70px;
+    }
+  }
+}
+
+/********************************FOOTER****************************/
+
+  footer {
+    width: auto;
+}
+
+  .footer-logo {
+    width: 400px;
+    height: 400px;
+}
+
+  .socialmedia-icons {
+    font-size: 30px;
+    padding-bottom: 50px;
+}
+
+  .socialmedia-icons li {
+    margin: 0 10px;
+}
+
+  .copyright-text {
+    font-size: 1.5rem;
+}
```

Comment:
Ska vara indenterat en tab in och } i slutet för media querien, därför inte den blir mindre i mobil.
På rad 50 har du måsvingen som ska vara i slutet av filen :)

Reply:
Nu ska måsvingen finnas och en tab in